src/client/src/services/executionService.js

```diff
@@ -39,7 +39,7 @@ export default class ExecutionService extends ServiceBase {
                   _this._serviceClient
                     .createRequestResponseOperation('executeTrade', executeTradeRequest)
                     .map(dto => {
-                      var trade = _this._tradeMapper.mapFromDto(dto.Trade);
+                      var trade = _this._tradeMapper.mapFromTradeDto(dto.Trade);
```

Comment:
find `var` and replace with `const` globally in vi :) or `let` though we should be using `const` for all immutables